### DISCUSSION TITLE
Assets are now one-click further

### DISCUSSION THREAD

**Discussion by spywhere** [community/community#6009] [Other Feature Feedback, Questions, & Ideas] [answered]
It would be nice to have a collapse section for Assets just like the old release UI. As this would be easier to download multiple release assets from multiple releases without having to go through each release.

**Comment by margual56** [comment]
Completely agree. 

Plus, and this is just an idea, what if all the releases' assets were collapsed except for the _newest_ release?

**Reply by serverwentdown** [reply]
A good idea!

**Comment by MylesBorins** [comment, marked as answer]
This came in https://github.com/github/feedback/discussions/5962#discussioncomment-1426476 and we already have a scoped work item to fix this.

Thank you for the feedback!

**Reply by MylesBorins** [reply]
as an update we shipped a fix this week
